docs/dev/dod.md

```diff
@@ -0,0 +1,16 @@
+# Definition of Done
+These criteria must be met before a feature or artifact can be considered "done".
+
+## Code
+- Code coverage: >75%
+- Codacy grade A
```

Comment:
Yeah you are right but you can disable the "weird excpecations". If some codacy issues cannot be solved or is not needed in the eyes of the team we can disable this check.